tox.ini

```diff
@@ -351,7 +351,7 @@ deps =
     -r{toxinidir}/requirements.txt
     func-s_base-py{35,36,37,38,39,310}: -r{toxinidir}/requirements_dev.txt
     pytest-mock
-    yea-wandb==0.7.28
+    git+https://github.com/wandb/yea-wandb.git@jq1/rename-queries#egg=yea-wandb
```

Comment:
before merging could you update `yea-wandb` with your new changes and pin to the relevant version? 
I could help publish it